### PR TITLE
Testing each FSA

### PR DESCRIPTION
**Pull request by seanrogers2657**
Just some simple tests for each FSA, more will be added later if needed/wanted.

### CODE REVIEW

**Comment by brianpmaher** [comment]
All of these methods could be private since they aren't being called anywhere else.. Not really suggesting a change.. I could handle it in the refactor ticket as well..

**Comment by bmaher91** [comment]
Pulled your branch noticed a

**Comment by brianpmaher** [comment]
+1

File: scanner.cs [hunk 1]
```diff
@@ -142,7 +142,7 @@ public class Scanner {
                     String.Format(Constants.ERROR_DISPATCHER_DIGIT, next)
                 );
             }
-        S1: // One or more digits have been read
+        S1: // One or more digits have been read\
```
Comment (JesseStephenBrown) [inline scanner.cs:145]: What's with the \ ?

**Pull request merged**